app/models/country.rb

```diff
@@ -138,11 +138,14 @@ def strata(year)
 
   def self.add_dump
```

Comment:
This now now uses the latest analysis year instead of a hard coded year. I'm not sure how this feature is used so it might need to be tweaked if/when used.